src/gui/skin_gothic.cpp

```diff
@@ -36,42 +30,70 @@ namespace REGoth
-    bs::HFont fontGothic20Book = loadFont("FONT_20_BOOK.FNT");
-    bs::HFont fontGothicOld10White = loadFont("FONT_OLD_10_WHITE.FNT");
+    bs::HFont fontGothicDefault      = loadFont("FONT_DEFAULT.FNT");
+    bs::HFont fontGothic10Book       = loadFont("FONT_10_BOOK.FNT");
+    bs::HFont fontGothic20Book       = loadFont("FONT_20_BOOK.FNT");
+    bs::HFont fontGothicOld10White   = loadFont("FONT_OLD_10_WHITE.FNT");
     bs::HFont fontGothicOld10WhiteHi = loadFont("FONT_OLD_10_WHITE_HI.FNT");
-    bs::HFont fontGothicOld20White = loadFont("FONT_OLD_20_WHITE.FNT");
+    bs::HFont fontGothicOld20White   = loadFont("FONT_OLD_20_WHITE.FNT");
     bs::HFont fontGothicOld20WhiteHi = loadFont("FONT_OLD_20_WHITE_HI.FNT");
 
     bs::GUIElementStyle baseStyle;
-    baseStyle.font = fontGothicDefault;
-    baseStyle.fontSize = 18;
-    baseStyle.wordWrap = true;
+    baseStyle.font             = fontGothicDefault;
+    baseStyle.fontSize         = 17;
+    baseStyle.wordWrap         = true;
+    baseStyle.normal.textColor = bs::Color::White;
     skin->setStyle("Label", baseStyle);
 
     bs::GUIElementStyle labelDefault = baseStyle;
-    labelDefault.font = fontGothicOld10White;
-    labelDefault.fontSize = 18;
+    labelDefault.font                = fontGothicOld10White;
+    labelDefault.fontSize            = 17;
     skin->setStyle("GothicLabel", labelDefault);
 
     bs::GUIElementStyle labelLarge = baseStyle;
-    labelLarge.font = fontGothicOld20White;
-    labelLarge.fontSize = 35;
+    labelLarge.font                = fontGothicOld20White;
+    labelLarge.fontSize            = 36;
     skin->setStyle("GothicLabelLarge", labelLarge);
 
     bs::GUIElementStyle labelDefaultHighlighted = baseStyle;
-    labelDefaultHighlighted.font = fontGothicOld10WhiteHi;
-    labelDefaultHighlighted.fontSize = 18;
+    labelDefaultHighlighted.font                = fontGothicOld10WhiteHi;
+    labelDefaultHighlighted.fontSize            = 17;
     skin->setStyle("GothicLabelHighlighted", labelDefaultHighlighted);
 
     bs::GUIElementStyle labelLargeHighlighted = baseStyle;
-    labelLargeHighlighted.font = fontGothicOld20WhiteHi;
-    labelLargeHighlighted.fontSize = 35;
+    labelLargeHighlighted.font                = fontGothicOld20WhiteHi;
+    labelLargeHighlighted.fontSize            = 36;
     skin->setStyle("GothicLabelLargeHighlighted", labelLargeHighlighted);
 
-    return skin;
+    bs::GUIElementStyle labelObjectFocus = baseStyle;
+    labelObjectFocus.font                = fontGothicOld10WhiteHi;
+    labelObjectFocus.fontSize            = 17;
+    labelObjectFocus.wordWrap            = false;
+    // labelObjectFocus.textHorzAlign = bs::TextHorzAlign::THA_Center;
+    // labelObjectFocus.textVertAlign = bs::TextVertAlign::TVA_Center;
+    skin->setStyle("GothicLabelObjectFocus", labelObjectFocus);
+
+    bs::GUIElementStyle subtitleBox = baseStyle;
+    subtitleBox.normal.texture      = gOriginalGameResources().sprite("DLG_CHOICE.TGA");
```

Comment:
You also load it in the constructor of UISubtitleBox. 
Is it not possible to use the style there instead of explicitly loading it?

Reply:
I guess this is obsolete, since we discard the style.

Reply:
I originally wanted to set the background image via the style but didn't get it to work. Do you know how that could work?

Reply:
By the way, the style will only be loaded once, since it's cached inside the load-method. I could rename the function to make this more obvious.

Reply:
Do you mean the texure is cached. Yes I did not think about that so it does not really matter. 
As for setting the background image via the style: 
I remember this(https://github.com/KirmesBude/REGoth-bs/blob/feature/console/src/components/Console/Console.cpp#L35 ) was working
Style [here](https://github.com/KirmesBude/REGoth-bs/blob/feature/console/src/gui/skin_gothic.cpp#L94)

Reply:
Ah, I thought you were talking about the multiple calls to `createSkin_Gothic`. But loading the texture is also cached.

Regarding the styles: I didn't know the `GUITexture` would use the texture given in the skin. Thanks for pointing it out!